clang/include/clang/Driver/Options.td

```diff
@@ -3465,7 +3465,7 @@ defm diagnostics_show_line_numbers : BoolFOption<"diagnostics-show-line-numbers"
   PosFlag<SetTrue>>;
 def fno_realloc_lhs : Flag<["-"], "fno-realloc-lhs">, Group<f_Group>,
   HelpText<"An allocatable left-hand side of an intrinsic assignment is assumed to be allocated and match the shape/type of the right-hand side">,
-  Visibility<[FlangOption, FC1Option]>;
+  Visibility<[ClangOption, CLOption, FlangOption, FC1Option]>;
```

Comment:
Would this now show up in `clang --help` and online guide? Is that what we want?
Is CLOption for Windows?

Reply:
> Would this now show up in `clang --help` and online guide? Is that what we want? Is CLOption for Windows?

Good point! Yes, `clang --help` now shows it, and I think we do not want this. I will check how I can resolve this. Any advices are welcome.

Yes, `CLOption` is for Windows driver.